bos

```diff
@@ -1062,6 +1062,7 @@ prog
   .help('--virtual invoices cannot be used with payers who probe before pay')
   .help('Only one --virtual invoice can be active at a time')
   .argument('[amount]', 'Amount for invoice', STRING, '0')
+  .option('--expiry <expires_in>', 'Hours the invoices expires in', INT)
```

Comment:
```suggestion
  .option('--hours <expires_in>', 'Hours the invoices expires in', INT)
```

What about change this to make it more direct what input to give it

Reply:
like adding --hours --minutes flags separately?

Reply:
In the future maybe, I think hours is enough for the basic functionality